app/Policies/AssignmentPolicy.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Assignment;
+use App\Models\User;
+use App\Models\Course;
+use Illuminate\Auth\Access\HandlesAuthorization;
+use Illuminate\Auth\Access\Response;
+
+class AssignmentPolicy
+{
+    use HandlesAuthorization;
+
+
+    /**
+    * Perform pre-authorization checks.
+    *
+    * @param  \App\Models\User  $user
+    * @param  string  $ability
+    * @return void|bool
+    */
+    public function before(User $user, $ability)
+    {
+        return $user->is_admin;
+    }
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAny(User $user)
+    {
+        return true;
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\Assignment  $assignment
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function view(User $user, Assignment $assignment)
+    {
+        return true;
+    }
```

Comment:
Should be determined based off of whether or not the user owns the assignment rather than always true